packages/labs/router/README.md

```diff
@@ -4,4 +4,193 @@ A router for Lit.
 
```

Comment:
Maybe add a disclaimer that this isn't yet released?

Unless we're planning not to merge this to main until it's ready to release (I think I'd advocate for merging even before we're ready for a preview release, so that we keep the tests running with mainline dev).